doc/sphinx-guides/source/developers/windows.rst

```diff
@@ -16,3 +16,91 @@ See the `post <https://groups.google.com/g/dataverse-dev/c/utqkZ7gYsf4/m/4IDtsvK
+
+Running Dataverse in Windows WSL
+--------------------------------
+
+It is possible to run Dataverse in Windows 10 and 11 through WSL (Windows subsystem for Linux)
+
+Install WSL
+~~~~~~~~~~~
+If you have Docker already installed, you should already have WSL installed, otherwise open PowerShell and run:
+
+.. code-block:: powershell
+  
+   wsl --install
+
+If you already had WSL installed you can install specific linux distribution:
+
+See the list of possible distributions:
+
+.. code-block:: powershell
+
+  wsl --list --online
+
+Choose the distribution you would like. Then run the following command. Notice that this installation of dataverse was tried with ubuntu distribution.
+
+.. code-block:: powershell
+
+  wsl --install -d <Distribution Name>
+
+You will be asked to create a linux user.
+After installation of Linux check that you have internet connection:
+
+.. code-block:: bash
+
+  ping www.google.com
+
+If you do not have internet connection try add in ``/etc/wsl.conf``
+
+.. code-block:: bash
+  
+  [network]
+  generateResolvConf = false
+
+Also in /etc/resolv.conf add
+
+.. code-block:: bash
+
+  nameserver 1.1.1.1
+
+Now you can install all the tools one usually uses in Linux. For example, it is good idea to run update:
+
+.. code-block:: bash
+
+   sudo apt update
+   sudo apt full-upgrade -y
+
+Install Dataverse
+~~~~~~~~~~~~~~~~~
+
+Now you can install Dataverse in WSL following the instructions for :doc:`classic-dev-env`
+At the end check that you have ``-Ddataverse.pid.default-provider=fake`` in jvm-options.
+
+Now you can access dataverse in your windows browser
+
+ - http://localhost:8080
+ - username: dataverseAdmin
+ - password: admin
+
+IDE for Dataverse in Windows
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Files in WSL are accessible from Windows for editing using ``\\wsl.localhost`` or ``\\wsl$`` path. Windows files are accessible in linux in ``/mnt/c/`` directory. Therefore one can use one's favorite editor or IDE to edit dataverse project files. Then one can build using ``mvn`` in WSL and deploy manually in WSL using ``asadmin``.
+
+It is still though possible to use full strength of IDE, the following instructions are for Intelij users.
+
+- Install Intelij in Windows.
+
+You can open the project through ``\\wsl.localhost`` and navigate to dataverse project.
+You can try to build the project in Intelij. You may get a message ``Cannot establish network connection from WSL to Windows host (could be blocked by firewall).`` In that case you can try
+to disable WSL Hyperviser from firewall.
+After that you should be able to build the project in Intelij.
+It seems that at present it is impossible to deploy the glassfish application in Intelij. You can try to add Glassfish plugin through Settings->Plugins and in Run->Edit Configurations configure Application Server from WSL ``/usr/localhost/payara6`` with URL http://localhost:8080 and Server Domain as domian one, but it may fail since Intelij confuses the Windows and Linux paths.
```

Comment:
```suggestion
It seems that at present it is impossible to deploy the Glassfish application in Intellij. You can try to add Glassfish plugin through Settings->Plugins and in Run->Edit Configurations configure Application Server from WSL ``/usr/localhost/payara6`` with URL http://localhost:8080 and Server Domain as domain1, but it may fail since Intellij confuses the Windows and Linux paths.
```